packages/blockchain-wallet-v4-frontend/src/scenes/plugin/common/FullScreenModal/FullScreenModal.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { animated, useTransition } from 'react-spring'
+import styled from 'styled-components'
+
+import ModalPortal from './ModalPortal'
+
+const ModalWrapper = styled.div`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 0;
+  top: 0;
+  box-sizing: border-box;
+  z-index: 1;
+`
+
+const ModalContent = styled.div`
+  background: ${(props) => props.theme.exchangeLogin};
+  position: absolute;
+  margin: auto;
+  width: 100%;
+  max-width: 800px;
+  height: 100%;
+  z-index: 1;
+  overflow: auto;
+`
+
+const InnerContent = styled.div`
+  padding: 20px;
+  height: 100%;
+  position: relative;
+  -webkit-overflow-scrolling: touch;
+`
+
+export interface IFullScreenModalProps {
+  isOpen: boolean
+  onClose?(): void
+}
+
+const FullScreenModal: React.FC<IFullScreenModalProps> = ({ children, isOpen }) => {
+  const contentRef = React.useRef(null)
```

Comment:
why do we need this?

Reply:
I probably forgot to remove while refactoring. I'll remove it.